### ISSUE TITLE
bug: Ionic SlidesPerView Render First Only One

### ISSUE THREAD

**Issue by danielptn** [ionic-team/ionic#21122]
# Bug Report

**Ionic version:**

[x] **4.x**

**Current behavior:**

When u use SlidesPerView > 1 in <ion-slides> component, when load the page, first show only one, after, show all the slides.. In my case, i have SlidesPerView="5", so, when i load the page, render the first one, and after 1 second, show all the image slide.

**Expected behavior:**

Load all in same time

**Steps to reproduce:**

Its easy to reproduce:

```
<ion-slides
    spaceBetween="10"
    slidesPerView="5"
  >

<ion-slide>
      <img [src]="some-img" />
</ion-slide>
<ion-slide>
      <img [src]="some-img" />
</ion-slide>
<ion-slide>
      <img [src]="some-img" />
</ion-slide>

</ion-slides>
```
With this, when load the page, note that the first one will render without others, and after some miliseconds, will render all.

**Comment by danielptn** [comment]
> Thanks for the issue! This issue has been labeled as `needs reproduction`. This label is added to issues that need a code reproduction.
> 
> Please provide a reproduction with the minimum amount of code required to reproduce the issue. Without a reliable code reproduction, it is unlikely we will be able to resolve the issue, leading to it being closed.
> 
> For a guide on how to create a good reproduction, see our [Contributing Guide](https://ionicframework.com/docs/contributing/how-to-contribute#creating-a-good-code-reproduction).

done.

**Comment by liamdebeasi** [comment]
Thanks for the follow up. `slidesPerView` is not a property on `ion-slides`. To set the number of slides per view, you need to use the `options` property.

```html
<ion-slides [options]="sliderOpts">
...
</ion-slides>
```

```javascript
sliderOpts = {
  slidesPerView: 5
}
```

For more information please see the [ion-slides Documentation](https://ionicframework.com/docs/api/slides).

**Comment by danielptn** [comment]
> Thanks for the follow up. `slidesPerView` is not a property on `ion-slides`. To set the number of slides per view, you need to use the `options` property.
> 
> ```
> <ion-slides [options]="sliderOpts">
> ...
> </ion-slides>
> ```
> 
> ```js
> sliderOpts = {
>   slidesPerView: 5
> }
> ```
> 
> For more information please see the [ion-slides Documentation](https://ionicframework.com/docs/api/slides).

This is not the problem. Even if it is placed like this, the problem persists.
Ever then i have more then one slide per view, the bug of render will appear.

**Comment by liamdebeasi** [comment]
Can you please provide a full reproduction of the issue via a GitHub repo? The issue is resolved for me when I set the `options` property.

**Comment by danielptn** [comment]
Good night, Liam, sorry, i try to use the option property, but i got this error:

> Can't bind to 'options' since it isn't a known property of 'ion-slides'

This is my Ionic Info:

> Ionic:
> 
>    Ionic CLI          : 5.4.13
>    Ionic Framework    : ionic-angular 3.9.2
>    @ionic/app-scripts : 3.2.4
> 
> Cordova:
> 
>    Cordova CLI       : 9.0.0 (cordova-lib@9.0.1)
>    Cordova Platforms : android 8.1.0, browser 6.0.0
>    Cordova Plugins   : cordova-plugin-ionic-webview 1.2.1, (and 10 other plugins)
> 
> Utility:
> 
>    cordova-res (update available: 0.13.0) : 0.10.0
>    native-run (update available: 1.0.0)   : 0.3.0
> 
> System:
> 
>    Android SDK Tools : 26.1.1 (C:\Users\danie\AppData\Local\Android\Sdk)
>    NodeJS            : v12.13.1 (C:\Program Files\nodejs\node.exe)
>    npm               : 6.12.1
>    OS                : Windows 10
>

**Comment by liamdebeasi** [comment]
Sorry, I did not realize you were using Ionic 3. This issue tracker is dedicated to Ionic 4+ bugs. You can open an issue on the [v3 repo](https://github.com/ionic-team/ionic-v3), but we are not actively supporting it anymore.

I would recommend posting on the [forums](http://forum.ionicframework.com/) to see if anyone else is running into a similar issue.
